common.sh

```diff
@@ -68,7 +70,7 @@ export RHCOS_IMAGE_NAME="redhat-coreos-maipo-${RHCOS_IMAGE_VERSION}"
 export RHCOS_IMAGE_FILENAME="${RHCOS_IMAGE_NAME}-qemu.qcow2"
 export RHCOS_IMAGE_FILENAME_OPENSTACK="${RHCOS_IMAGE_NAME}-openstack.qcow2"
 export RHCOS_IMAGE_FILENAME_DUALDHCP="${RHCOS_IMAGE_NAME}-dualdhcp.qcow2"
-export RHCOS_IMAGE_FILENAME_LATEST="redhat-coreos-maipo-latest.qcow2"
+export RHCOS_IMAGE_FILENAME_LATEST="${RHCOS_IMAGE_NAME}-latest.qcow2"
```

Comment:
$RHCOS_IMAGE_FILENAME_LATEST  shouldn't contain the version number, we created it so that we would have a versionless link to point at.

Reply:
It still doesn't have the version.  That variable contains the base of the name, without the version number.  I updated this to keep it consistent with other images.  The naming convention changed in the most recent images.

Reply:
I'm seeing it in the logs with a version number

```
++ RHCOS_IMAGE_NAME=rhcos-maipo-400.7.20190312.0
++ RHCOS_IMAGE_FILENAME_LATEST=rhcos-maipo-400.7.20190312.0-latest.qcow2
```

Reply:
Ah, ooops :-)

Reply:
I changed this back to a static filename, but using the latest naming convention